inversion-c/test-inversion.c

```diff
@@ -0,0 +1,42 @@
+#include "inversion.c"
+#include <string.h>
+#include <time.h>
+
+int main() {
+  uint64_t res[LIMBS], out[LIMBS], g[LIMBS], g2[LIMBS], g3[LIMBS];
+  uint8_t a[BYTES];
+  
+  srand(time(0));
+  for (int j = 0; j < 1000; j++) {
+    int i;
+    for (i = 0; i < BYTES; i++) {
+      a[i] = rand() % 256;
+      if (i > BYTES - 8) a[i] = 0;
+    }
+    
+    fiat_test_from_bytes(g,a);
+    fiat_test_from_bytes(g2,a);
+    fiat_test_from_montgomery(g3,g2);
+
+    inverse(out,g3);
+    
+    fiat_test_mul(res,out,g);
+    fiat_test_from_montgomery(out,res);
+    fiat_test_to_bytes(a,out);
+
+    if (a[0] != 1) {
+      printf("FAIL\n");
+      return 2;
+    }    
+    for (i = 1; i < BYTES; i++) {
+      if (a[i] != 0) {
+	printf("FAIL\n");
+      	return 1;
```

Comment:
Please use spaces rather than tabs
```suggestion
        printf("FAIL\n");
        return 1;
```